test/package.json

```diff
@@ -27,7 +27,7 @@
     "bedrock-package-manager": "^1.0.1",
     "bedrock-passport": "^6.0.0",
     "bedrock-permission": "^3.2.0",
-    "bedrock-profile": "^7.1.0",
+    "bedrock-profile": "github:digitalbazaar/bedrock-profile#issue-30",
```

Comment:
same issue here. We need a release not a branch in package.json.

Reply:
Updated 2dd6e9e